### PR TITLE
Publish new version of dataset to Zenodo automatically

### PR DESCRIPTION
**Pull request by jdddog**

### CODE REVIEW

File: academic_observatory_workflows/workflows/oa_web_workflow.py [hunk 1]
```diff
@@ -1237,6 +1353,8 @@ def __init__(
         ror_dataset_id: str = "ror",
         settings_dataset_id: str = "settings",
         version: str = "v3",
+        conceptrecid: int = 6399462,
+        zenodo_host: str = "https://zenodo.org",
```
Comment (aroelo) [inline academic_observatory_workflows/workflows/oa_web_workflow.py:1357]: Could you add descriptions for these params to the docstring just below? It is a little bit unclear what the 'conceptrecid' is

**Review by aroelo — CHANGES_REQUESTED** [review]
Hi @jdddog, thanks it looks good.
I have one small comment on the added attributes to the OaWebWorkflow class.
Also, because the Zenodo class is mocked during the unit test for the complete workflow, it is not covered by any tests. It might be useful to add separate unit tests for this class.

File: academic_observatory_workflows/workflows/oa_web_workflow.py (hunk 1 above)
Comment (jdddog) [reply academic_observatory_workflows/workflows/oa_web_workflow.py:1357]: Thanks Aniek, yeah sure.

**Review by aroelo — APPROVED** [review]
Thanks @jdddog!

**Pull request merged**